cmake/bdwgc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/gc_cpp.cc b/gc_cpp.cc
+index 341805a6..fb6b0a01 100644
+--- a/gc_cpp.cc
++++ b/gc_cpp.cc
+@@ -38,7 +38,7 @@ built-in "new" and "delete".
+ #endif
+ #include "gc_cpp.h"
+ 
+-#if !(defined(_MSC_VER) || defined(__DMC__)) || defined(GC_NO_INLINE_STD_NEW)
++#if !defined(SKIP_CPP_DEFINITIONS) && (!(defined(_MSC_VER) || defined(__DMC__)) || defined(GC_NO_INLINE_STD_NEW))
```

Comment:
It might be worthwhile to upstream this. This is the only modification need to be able to link BDWGC statically.

Reply:
https://github.com/ivmai/bdwgc/pull/643